service/src/main/java/org/apache/griffin/core/job/SparkSubmitJob.java

```diff
@@ -204,6 +204,7 @@ private void setLivyArgs() throws IOException {
         args.add(finalMeasureJson);
         args.add("raw,raw");
         livyConfMap.put("args", args);
+        livyConfMap.remove("name");
```

Comment:
I agree to remove duplicated names. 
But how should we handle the cases when users want to explicitly specify name by their job config?

Reply:
@guoyuepeng 
Instead of hard coding，we can explicitly specify the livy session name in the  configuration file. which is null by default.

Or we can use the DQ job name to be the livy session name?

Reply:
Agree, 
we can use 
name_specified in config file | session_name